src/nvim/os/os_win_conpty.c

```diff
@@ -0,0 +1,181 @@
+#include <uv.h>
```

Comment:
not sure if PVS studio works at source level but maybe add [the header](https://github.com/neovim/neovim/blob/21fdfc87284d4f58bc17883429b0a152013c618a/src/nvim/os/tty.c#L1) anyways.